python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -690,6 +724,8 @@ def slice_skip(tbl: plc.Table) -> plc.Table:
                 null_order=plc.types.NullOrder.AFTER,
                 name=name,
             )
+            # TODO: Preserve the selection order eg. col then index col
+            # which we currently do not do
```

Comment:
Can you add a test for this please?

Reply:
It was simple enough to support it in this PR. Added a test too in https://github.com/rapidsai/cudf/pull/18581/commits/273ea91d1809922480870081d5c190027dce1909